mirgecom/initializers.py

```diff
@@ -807,3 +808,77 @@ def exact_rhs(self, discr, q, t=0.0):
 
         return join_conserved(dim=self._dim, mass=massrhs, energy=energyrhs,
                               momentum=momrhs, species_mass=yrhs)
+
+
+class MixtureInitializer:
+    r"""Solution initializer for multi-species mixture.
```

Comment:
What distinguishes this from, e.g. the `MulticomponentLump`?

Reply:
6132569